libctru/include/3ds/svc.h

```diff
@@ -241,6 +241,30 @@ typedef struct {
 
 ///@}
 
+///@name Processes
+///@{
+
+/// Information on address space for process. All sizes are in pages (0x1000 bytes)
+typedef struct {
+	u8 name[8];		///< ASCII name of codeset
+	u16 unk1;
+	u16 unk2;
+	u32 unk3;
+	u32 text_addr;				///< .text start address
```

Comment:
I was going to do that, but it adds confusion to the _total members. My code is like this
```
info.ro_addr = ...;
info.ro_size = ...;
info.ro_size_total = ...;
```
Since the proposed struct doesn't seem to be used anywhere else, I think it's cleaner to leave it as is.